packages/vega-parser/src/parsers/guides/axis-domain.js

```diff
@@ -25,18 +27,27 @@ export default function(spec, config, userEncode, dataRef) {
     strokeOpacity:    _('domainOpacity')
   });
 
-  if (orient === Top || orient === Bottom) {
-    u = 'x';
-    v = 'y';
+  if (isSignal(spec.orient)) {
+    enter.y = xyAxisConditionalEncoding('x', orient.signal, zero, position(spec, 0));
```

Comment:
Why is this using 'x' for `enter.y`?

Reply:
Why should `enter.y` be different from `update.y` while they are the same in the non-signal condition?

Reply:
Rewritten to make this logic more clear (now matches the pattern in the non-signal case).

Reply:
(Matched in other files)